packages/api/migrations/20240329101306-add-exercice-field-if-not-exists.js

```diff
@@ -5,18 +5,11 @@ module.exports = {
         await db
             .collection(COLLECTION)
             .aggregate([
-                { $match: { exercice: { $exists: false } } },
                 {
-                    $addFields: {
-                        exercice: { $year: "$conventionDate" },
-                    },
+                    $addFields: { exercice: { $ifNull: ["$exercice", { $year: "$conventionDate" }] } },
```

Comment:
```suggestion
                    $addFields: { exercice: { $ifNull: [{ $year: "$conventionDate" }, "$exercice"] } },
```
C'est dans ce sens non ?

Reply:
non c'est `[...[champ que tu veux tester], valeur par laquelle changer]` donc ici on remplace `exercice` quand `null` par l'année de $conventionDate

Reply:
https://www.mongodb.com/docs/manual/reference/operator/aggregation/ifNull/